tools/nl/loadtest/loadtest_results.csv

```diff
@@ -0,0 +1,43 @@
+Result,Query,NumCharts,ChartTypes,Details

```

Comment:
Should save this as a json (it does not feel like a csv...)

Reply:
Ok, let me open that up...